ubcpi/static/js/src/translations.js

```diff
@@ -1,7 +1,18 @@
+    gettextCatalog.setStrings('ru', {"Error:":"Ошибка:","Problem Weight":"Вес задания"});
+    gettextCatalog.setStrings('zh_CN', {});
```

Comment:
Any reason why the Simplified Chinese translations are removed?

Reply:
The Transifex resources was not reviewed. Now they're and can be pulled.